cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,12 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
-
+    cy.visit('https://react-redux.realworld.io');
+    cy.get(':nth-child(2) > .nav-link').click();
```

Comment:
Using a CSS selector like ':nth-child(2) > .nav-link' is not recommended as it may lead to brittle tests. If the structure of the DOM changes, this test could fail. Consider using data-test-id attribute or more specific class or id selector.